features/src/python-lit/install.sh

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env bash
+set -ex
+
+LIT_VERSION="${VERSION:-latest}";
+
+if [ "$LIT_VERSION" -neq "latest" ]; then
+  LIT_VERSION_TO_INSTALL="lit==$LIT_VERSION"
+else
+  LIT_VERSION_TO_INSTALL="lit"
+fi;
+
+# Ensure we're in this feature's directory during build
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )";
+
+# install global/common scripts
+. ./common/install.sh;
+
+
+check_packages                  \
+    gpg                         \
+    dirmngr                     \
+    apt-utils                   \
+    gettext-base                \
+    bash-completion             \
+    gcc g++ gfortran            \
+    software-properties-common  \
+    ;
```

Comment:
Does `lit` need all these dependencies, or are these needed in order to build/install lit/wheel/psutil etc.?